report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate of the device, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
+Therefore, a test of the motors has been conducted, checking their precision and quality.
+
+\subsection*{Motors}
+To ensure the capabilities of the motors, the tests have to try out combinations of target angles and movement speed, and to what extent precision is reached, considering the braking distance.
+Therefore a setup combining an angle, and a velocity ranging 0-100 is setup as seen in Appendix~\ref{appendix:motortest}.
```

Comment:
"Ranging 0-100" giver ikke mening for mig... (eller jo men det lyder forkert)
"ranging from 0 to 100"
eller "in range 0 to 100" (eller 0-100)